db/migrate/20161222111649_create_issues.rb

```diff
@@ -0,0 +1,20 @@
+class CreateIssues < ActiveRecord::Migration[5.0]
+  def change
+    create_table :issues do |t|
+      # belongs to
+      t.integer :reporter_id, null: false
```

Comment:
This field should be added when it will be needed. Please avoid of making things that are not needed at the moment.